### PR TITLE
Documented has_and_belongs_to_many relationships.

### PR DESCRIPTION
**Pull request by ecnalyr**
- There was a lack of documentation for how to create factories with
models that have has_and_belongs_to_many relationships.
- An example is provided to show how to accomplish this.
- This functionality was already present, this is only a documentation
update.

### CODE REVIEW

**Comment by tute** [comment]
This looks good to me. cc @joshuaclayton

**Comment by greggawatt** [comment]
Is there a reason this hasn't been merged in yet?

**Comment by ecnalyr** [comment]
I know of no reason.

**Comment by tute** [comment]
@joshuaclayton, want to merge this, or leave feedback? Thank you!

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -366,6 +366,51 @@ create(:user_with_posts).posts.length # 5
 create(:user_with_posts, posts_count: 15).posts.length # 15
 ```
 
+Generating data for a `has_and_belongs_to_many` relationship is very similar to the above `has_many` relationship, with a small change, you need to pass an array of objects to the model's pluralized attribute name rather than a single object to the singular version of the attribute name.
```
Comment (joshuaclayton) [inline GETTING_STARTED.md]: can we line break at 80 chars?

**Comment by ecnalyr** [comment]
@joshuaclayton 

> can we line break at 80 chars?

Done.

**Comment by joshuaclayton** [comment]
@ecnalyr all set, thanks! I've merged this as 2567a51ebd92377977e184c99e365c60b86111b9

**Pull request closed without merging**